pkg/path/finch.go

```diff
@@ -60,6 +61,11 @@ func (w Finch) LimaSSHPrivateKeyPath() string {
 	return fmt.Sprintf("%s/lima/data/_config/user", w)
 }
 
+func (w Finch) generatePathSum() string {
```

Comment:
So the install location in this case is the directory two levels up from the finch executable, right? I think this works for now, but we might have to revisit this to somehow include the instance name if we ever see a usecase for multiple finch instances (unless we want them to share their data disks?)

Reply:
> we might have to revisit this to somehow include the instance name if we ever see a usecase for multiple finch instances

yeah, I figure we'll need to overhaul the `path` package pretty significantly in order to do that anyway. the disk name is pretty arbitrary though so I might as well do it based off of `.../lima/data/finch` to "future-proof" it

> unless we want them to share their data disks?

this won't be possible based on the current implementation of persistent disks, if two instances try to write to the same qcow2 disk at the same time the disk will corrupt and very strange things happen. probably the best we could do is copy the other instance's disk at `finch vm init`